src/Notification.ts

```diff
@@ -11,11 +11,11 @@ export class Notification<T> {
   observe(observer: Observer<T>): any {
```

Comment:
Hmm... I thought I also had a PR to change the Observer interface to allow partials as well...